portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+/* toggles the drop down text */
+function toggleDropDown(id) {
+  document.getElementById(id).classList.toggle("show");
+}
+
+/* display the content corresponding to the tab-button clicked */
```

Comment:
Use the verb form, i.e. "Displays" for all the functions.

Good job on the previous one where you said "Toggles"

Reply:
Done